Examples/MAX78002/CSI2/main.c

```diff
@@ -195,7 +210,7 @@ void service_console()
     clear_serial_buffer();
 }
 
-volatile int buttonPressed = 0;
```

Comment:
Be careful taking out the volatile, may cause example to hang if compiler optimization level is increased.

Reply:
Thanks, good catch